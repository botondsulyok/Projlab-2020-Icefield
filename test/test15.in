field spawn 0
field unstable 1 false 0
neighbours spawn unstable
neighbours unstable spawn
entity mover Eskimo spawn
entity falsepuller Eskimo spawn
stats spawn
stats unstable
stats mover
stats falsepuller
move mover unstable
stats spawn
stats unstable
stats mover
stats falsepuller