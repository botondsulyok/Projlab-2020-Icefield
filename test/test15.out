id: spawn
type: iceunit
building: plains0
snow: 0
pickable: null
standers: mover, falsepuller
pullers: mover, falsepuller
masters: mover, falsepuller
neighbours: unstable

id: unstable
type: unstable
building: plains1
snow: 0
pickable: null
standers: null
pullers: null
masters: null
neighbours: spawn
capacity: 1
broken: false

id: mover
type: eskimo
iceUnit: spawn
movebehaviour: walk0
bodyheat: 5
stamina: 4
digger: barehand0
waterProtecter: lung0
pullerDevice: basicpull0
tent: null
components: null

id: falsepuller
type: eskimo
iceUnit: spawn
movebehaviour: walk1
bodyheat: 5
stamina: 4
digger: barehand1
waterProtecter: lung1
pullerDevice: basicpull1
tent: null
components: null

Game Over
id: spawn
type: iceunit
building: plains0
snow: 0
pickable: null
standers: falsepuller
pullers: falsepuller
masters: falsepuller
neighbours: unstable

id: unstable
type: unstable
building: plains2
snow: 0
pickable: null
standers: mover
pullers: mover
masters: mover
neighbours: spawn
capacity: 1
broken: true

id: mover
type: eskimo
iceUnit: unstable
movebehaviour: walk0
bodyheat: 5
stamina: 3
digger: barehand0
waterProtecter: lung0
pullerDevice: basicpull0
tent: null
components: null

id: falsepuller
type: eskimo
iceUnit: spawn
movebehaviour: walk1
bodyheat: 5
stamina: 4
digger: barehand1
waterProtecter: lung1
pullerDevice: basicpull1
tent: null
components: null

